ruby/lib/cucumber/ci_environment/variable_expression.rb

```diff
@@ -1,24 +1,29 @@
+# frozen_string_literal: true
+
 module Cucumber
   module CiEnvironment
     module VariableExpression
       def evaluate(expression, env)
         return nil if expression.nil?
+
         begin
           expression.gsub(/\${(.*?)(?:(?<!\\)\/(.*)\/(.*))?}/) do
-            variable = $1
-            pattern = $2
-            replacement = $3
+            variable = ::Regexp.last_match(1)
```

Comment:
Not sure these need to be top-level namespaced

Reply:
Tests pass when refactoring `::Regexp` to just `Regexp` do you want me to change these to `Regexp`?